torchvision/transforms.py

```diff
@@ -124,7 +124,7 @@ class Scale(object):
     """
 
     def __init__(self, size, interpolation=Image.BILINEAR):
-        assert isinstance(size, int) or (isinstance(size, tuple) and len(size) == 2)
+        assert isinstance(size, int) or ((isinstance(size, tuple) or isinstance(size, list)) and len(size) == 2)
```

Comment:
nit: maybe it would be simpler to have a `isinstance(size, collections.Iterable)` instead of `(isinstance(size, tuple) or isinstance(size, list)) `?